DataConnectors/Imperva WAF Gateway/Logo/Imperva_DarkGrey_final.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Logo requirement:
Logo should not have ids and href.
Refer: https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md
a. Logo needs to be in SVG format and under 5 Kb
b. Ensure raw file of logo does not have any of the following:
o cls and style formats
o embedded png formats
o xmlns:xlink
o data-name
c. Do not use xlink:href - use inline instead
d. Do not use title tag
e. If some properties in the logo have IDs (for e.g. <g id="layer0"...), then set these IDs as GUIDs so that these are uniquely identifiable across all Azure logo assets
f. Render correctly in 75pt*75pt square box.
 
I don't know much about svg file, but based on the requirement, I gave a try and modified the logo according to the requirements.

Plus, the logo is required to be rendered correctly in 75pt * 75pt view box, even with my changes, it doesn't meet the requirement yet. I think you might need to adjust the vector value and test it visually.

This is something you could start with, but again please fix the 75 * 75 requirement before check this in.

<svg width="952px" height="424px" viewBox="0 0 952 424" version="1.1">
<g stroke="none" fill="#212121" stroke-width="1" fill-rule="evenodd" transform="translate(115.500000, 113.000000)" >
<path d="M275.733402 42.4363752 C306.544017 42.4363752 324.082278 68.0921935 324.082278 98.0239815 C324.082278 127.956924 306.544017 153.374868 275.733402 153.374868 C264.358416 153.374868 253.931537 147.91068 249.190999 142.212081 L249.190999 194.528334 L219.804274 194.528334 L219.804274 44.812812 L249.190999 44.812812 L249.190999 53.6026261 C253.931537 47.9005633 264.358416 42.4363752 275.733402 42.4363752 Z M385.101004 42.4363752 C415.910467 42.4363752 431.788676 63.8185332 431.788676 95.1729501 L431.788676 104.677543 L363.532998 104.677543 C365.903843 122.019296 378.465404 129.146297 394.106298 129.146297 C405.718599 129.146297 414.015404 126.530831 424.442283 119.879579 L424.442283 144.109305 C415.672 150.523837 403.823536 153.374868 389.605379 153.374868 C357.372027 153.374868 335.331695 132.469614 335.331695 98.7399149 C335.331695 65.4790367 356.186605 42.4363752 385.101004 42.4363752 Z M672.302706 42.4363752 C701.453269 42.4363752 718.988074 57.4040013 718.988074 83.2965395 L718.988074 150.999586 L689.600196 150.999586 L689.600196 145.298678 C686.046233 148.863333 675.620506 152.901428 664.008205 152.901428 C642.678666 152.901428 624.900785 140.549268 624.900785 118.454641 C624.900785 98.2618561 642.678666 84.4847579 665.903268 84.4847579 C675.144725 84.4847579 685.336593 87.5713545 689.600196 90.6602605 L689.600196 84.7214778 C689.600196 74.5078802 680.83337 68.0921935 665.903268 68.0921935 C655.237922 68.0921935 645.284521 71.1822542 636.990021 76.6452876 L636.990021 51.2261892 C644.573729 46.475625 658.317256 42.4363752 672.302706 42.4363752 Z M548.238861 44.812812 L572.650038 115.365735 L596.587737 44.812812 L626.211777 44.812812 L585.921239 150.999586 L557.483773 150.999586 L517.431702 44.812812 L548.238861 44.812812 Z M500.67681 42.4363752 C505.653511 42.4363752 510.156734 43.3890284 511.815634 44.3382175 L511.815634 73.795411 C507.787041 71.4189741 502.574177 70.2307557 497.122847 70.2307557 C487.169446 70.2307557 479.111108 75.458224 476.97873 85.1983818 L476.97873 150.999586 L447.589701 150.999586 L447.589701 44.812812 L476.97873 44.812812 L476.97873 55.2654391 C481.480801 47.6638434 490.249932 42.4363752 500.67681 42.4363752 Z M166.084708 42.4363752 C187.888878 42.4363752 200.921612 57.6407212 200.921612 81.6337265 L200.921612 150.999586 L171.533735 150.999586 L171.533735 87.8103838 C171.533735 76.407413 166.557034 68.8069721 156.36747 68.8069721 C148.784914 68.8069721 141.910846 73.5563816 140.014631 82.5852251 L140.014631 150.999586 L110.391743 150.999586 L110.391743 87.8103838 C110.391743 76.407413 105.648901 68.8069721 95.4593372 68.8069721 C87.8756289 68.8069721 80.7653985 73.5563816 78.8703354 82.5852251 L78.8703354 150.999586 L49.4847619 150.999586 L49.4847619 44.812812 L78.8703354 44.812812 L78.8703354 53.6026261 C84.0855028 46.9513743 93.800437 42.4363752 105.648901 42.4363752 C118.210462 42.4363752 128.163863 48.1395927 134.088671 55.9779082 C140.962739 48.3774673 151.152303 42.4363752 166.084708 42.4363752 Z M30.1919833 44.812812 L30.1919833 150.999586 L0.806409809 150.999586 L0.806409809 44.812812 L30.1919833 44.812812 Z M670.879969 105.152137 C661.63736 105.152137 652.157437 109.190232 652.157437 118.216766 C652.157437 127.481175 661.63736 131.28255 670.879969 131.28255 C679.17447 131.28255 686.995493 127.720204 689.600196 122.019296 L689.600196 114.415391 C686.995493 108.716793 679.17447 105.152137 670.879969 105.152137 Z M269.100105 68.0921935 C260.330975 68.0921935 252.746114 73.3196617 249.190999 80.4466628 L249.190999 115.365735 C252.746114 122.729456 260.567138 127.720204 269.100105 127.720204 C284.978315 127.720204 294.457086 114.89114 294.457086 98.0239815 C294.457086 80.9201026 284.978315 68.0921935 269.100105 68.0921935 Z M718.988074 165.011094 L718.988074 194.465978 L689.600196 194.465978 L689.600196 165.011094 L718.988074 165.011094 Z M385.811797 65.954786 C397.424098 65.954786 403.823536 74.0321309 404.535481 85.4351017 L404.535481 85.4351017 L364.481106 85.4351017 C367.798906 72.132598 376.095711 65.954786 385.811797 65.954786 Z M30.1919833 1.34641937 L30.1919833 30.8013034 L0.806409809 30.8013034 L0.806409809 1.34641937 L30.1919833 1.34641937 Z">
</path>
</g>
</svg>